app/src/main/res/drawable/ic_help_grey_icon_24dp.xml

```diff
@@ -1,6 +1,7 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
   android:width="24dp"
   android:height="24dp"
+  android:autoMirrored="true"
```

Comment:
This should not be reversed. Only icons which represent direction should be reversed.

Reply:
Reverted.